.golangci.yml

```diff
@@ -7,9 +7,16 @@ linters:
   disable:
     - dupl
     - gochecknoglobals
-    - wsl
-    - lll
+    - godot
```

Comment:
Nit: Some comments here to explain why each of these linters is too noisy helps with future evaluations, if you've got it handy/ fresh in your mind.